vizro-core/changelog.d/20230918_111226_petar_pejovic_optimise_app_rendering_time.md

```diff
@@ -0,0 +1,43 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category.
+
+-->
+<!--
+
+### Changed
+
+- Optimise the client-server communication ([#34](https://github.com/mckinsey/vizro/pull/34))
+
+-->
```

Comment:
```suggestion

### Changed

- Optimise the client-server communication ([#34](https://github.com/mckinsey/vizro/pull/34))
    - eliminate most server side callback in favour of client-side callbacks
    - logging information now only displayed for action function carried out (no trigger or finished information)

```

Reply:
Do not forget the comment out, and also to check whether my indentations actually play out nicely